packages/mux-player/src/media-theme-mux/media-theme-mux.ts

```diff
@@ -85,7 +88,6 @@ export default class MediaThemeMux extends MediaTheme {
           exportparts="layer, media-layer, poster-layer, vertical-layer, centered-layer"
         >
           <slot name="media" slot="media"></slot>
-          <media-poster-image slot="poster" src="${props.poster}"></media-poster-image>
```

Comment:
blocking: shouldn't be removed right?

Reply:
oh man good catch. Not sure why that happened with the rebase merge.